### ISSUE TITLE
Windows - crash with Access Violation in MSVCR120.dll

### ISSUE THREAD

**Issue by darkphoenix** [processone/ejabberd#3123]
In a working Windows install of ejabberd 19.02, a few months ago crashes on startup kept occurring supposedly caused by an Access Violation (0xc0000005) in MSVCR120.dll. An update install of 19.09.1 ended with an install script failure (also with an erl.exe crash) and the same issues still occurring. As I understand, ejabberd ships its own Erlang and MSVCR120.dll, so the same error occurring with both installs is rather confusing to me, along with the fact that removing the MSVCR120.dll to fall back to the OS version did not fix the error either. Process Explorer claims the bundled MSVCR120.dll is actually the one being loaded when it is present. I am quite frankly at wit's end at this point as I don't have the means to debug an error in native code and there is no helpful error log, so I'm opening this issue. What might be causing this problem?

**Comment by swifty94** [comment]
Greetings all,
I've faced a very similar issue and opened a separate ticket, however, let me attach my details here as well in case it will help to speed up the process and find the root cause and especially solution.

---------------------------------------------------------------------------------
Environment

    ejabberd version: 20.01
    Erlang version:
    Erlang/OTP 22 [erts-10.5] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1]
    Eshell V10.5 (abort with ^G)
    OS: Windows Server 2012 R2 64bit
    Installed from: official website https://www.process-one.net/en/ejabberd/downloads/

Installation hangs and fails on the step of post-install script execution.

Screenshot attached.
![ejabberd_install_fail](https://user-images.githubusercontent.com/49090543/74327035-246f6480-4d94-11ea-8d50-87e29a8cc08c.png)

Details about the error below:

Problem signature:
Problem Event Name: APPCRASH
Application Name: erl.exe
Application Version: 0.0.0.0
Application Timestamp: 5c86ce75
Fault Module Name: MSVCR120.dll
Fault Module Version: 12.0.21005.1
Fault Module Timestamp: 524f83ff
Exception Code: c0000005
Exception Offset: 000000000003c798
OS Version: 6.3.9600.2.0.0.16.79
Locale ID: 9226
Additional Information 1: e91a
Additional Information 2: e91a6bd26807196ed21cb7c46c0b76ec
Additional Information 3: 8924
Additional Information 4: 892491a9552acccdeb582488560d0d7b

Read our privacy statement online:
http://go.microsoft.com/fwlink/?linkid=280262


------------------------------------------------------------------
I've installed the update for Visual C++ 2013 and Visual C++ Redistributable Package already (judging on MSVCR120.dll mentioning in error code and some researching in Google / GitHub), but no luck. I am facing exactly the same error code and I am not able to install the EjabberD at all.
